docs/dotnet/TOC.yml

```
@@ -8,7 +8,7 @@ items:
 - name: MVVM Toolkit
   items:
     - name: Introduction
-      href: /windows/communitytoolkit/mvvm/introduction
+      href: /mvvm/introduction
```

```suggestion
      href: mvvm/introduction
```

The initial slash roots it, so you need to remove it to do the relative path or do `/dotnet/communitytoolkit/dotnet/mvvm`

Also check the index.yml and toc.yml files in the next folder up as well. 😉

I forgot about creating the folder within the docs here and how that'll double-`dotnet` us in urls:

https://docs.microsoft.com/en-us/dotnet/communitytoolkit/dotnet

Did we want it that way? I think if we want to flatten we'd have a folder in the docs for each sub-package like `mvvm`, `high-performance`, and `diagnostics`. Though I don't think we could have a separate TOC that way 🙁

Thoughts? (We just have the one index page and its new, so we'll want to fix this now over later.

---

Ah, whoops! Fixed, thank you! 😄

---

Thoughts on the doc url duplication?

---

We discussed with @brminnick and decided to flatten the urls down. I've updated the PR for that so we'll test out how it works. I also opened issue #54 as I think we need to fix the breadcrumbs file somehow too, but I don't know how that works yet.